pages/_app.js

```diff
@@ -3,22 +3,19 @@ import Head from "next/head";
 import { useRouter } from "next/router";
 import { useState } from "react";
 import { useProjectsList } from "../components/custom-hooks/useProjectsList";
-import { mockProjects } from "../db/projects";
-import { uid } from "uid";
-import dayjs from "dayjs";
 
 export default function App({ Component, pageProps }) {
   const [projectName, setProjectName] = useState("");
 
-  const [projectStatus, setProjectStatus] = useState("");
-  const [feeling, setFeeling] = useState("");
+  const [newProjectStatus, setNewProjectStatus] = useState("");
+  const [newProjectFeeling, setNewProjectFeeling] = useState("");
 
   const router = useRouter();
 
   const { handleAddProjectSubmit, projectsList } = useProjectsList(
     projectName,
-    projectStatus,
-    feeling
+    newProjectStatus,
+    newProjectFeeling
   );
 
   function handlePreAddSubmit(e) {
```

Comment:
Same here (line 21 to 34) as i mentioned in the `index.js`: For better readability we would like to avoid shorthand notation.